### PR TITLE
License and copyright notice

### PR DESCRIPTION
**Pull request by Honny1**
This PR adds license and copyright notice according to [SPDX Specification](https://spdx.dev/).

### CODE REVIEW

File: openscap_report/cli.py [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# Copyright 2022, Red Hat, Inc.
+# SPDX-License-Identifier: LGPL-2.1
```
Comment (matejak) [inline openscap_report/cli.py:2]: GPL would be a better fit, as this is the script that calls the rest of the product that is a LGPL-licensed module.

**Review by matejak — CHANGES_REQUESTED** [review]
I just recommend to add a GPL licence one and license the entrypoint.

File: openscap_report/html_report/templates/css/patternfly.css [hunk 2]
```diff
@@ -1,3 +1,8 @@
+/*!
```
Comment (evgenyz) [inline openscap_report/html_report/templates/css/patternfly.css:1]: Are you going to update this file manually every time you update PF? I'd say that if you want to add license information for an external file, it's better to have it as a separate file. Like `patternfly-license.txt` or something like that.

File: tests/__init__.py [hunk 3]
```diff
@@ -0,0 +1,2 @@
+# Copyright 2022, Red Hat, Inc.
```
Comment (evgenyz) [inline tests/__init__.py:1]: I don't think that an empty file deserves that.

File: openscap-report.spec [hunk 4]
```diff
@@ -6,7 +6,8 @@ Version:        0.0.0
 Release:        0%{?dist}
 Summary:        A tool for generating human-readable reports from (SCAP) XCCDF and ARF results
 
-License:        LGPL-2.1
+# The entire source code is LGPL-2.1+ and GPL-2.0+ except schemas/ and assets/, which are Public Domain
+License:        (LGPLv2+ or GPLv2+) and Public Domain
```
Comment (matejak) [inline openscap-report.spec:10]: Is the possibility of using logical expression in this field documented anywhere? I wasn't able to find it by looking at https://rpm-packaging-guide.github.io/ and https://fedoraproject.org/wiki/Licensing:Main

Comment (Honny1) [reply openscap-report.spec:10]: Here: https://docs.fedoraproject.org/en-US/packaging-guidelines/LicensingGuidelines/#_multiple_licensing_scenarios

**Review by evgenyz — APPROVED** [review]

**Pull request merged**